doc/releasenotes/17_0_0_summary.md

```diff
@@ -3,11 +3,26 @@
 ### Table of Contents
 
```

Comment:
just wondering do you wanna talk about the newly added metrics here as well?

Reply:
just like you have given example in website doc.... just floating an idea here

Reply:
Added some more details!